test/fixtures/vcr_cassettes/actions/pulp3/ansible_collection_sync/sync.yml

```diff
@@ -43,7 +51,11 @@ http_interactions:
       Cross-Origin-Opener-Policy:
       - same-origin
       Correlation-Id:
+<<<<<<< HEAD
       - 860cc24bec6e4aa8ad51bb36e23a67de
+=======
```

Comment:
The VCRs seem to have merge conflicts of some sort..What you can do is restore these files, rebase and then re-record the sync test..